java/src/lib.rs

```diff
@@ -138,6 +140,35 @@ pub extern "system" fn Java_glide_ffi_resolvers_RedisValueResolver_valueFromPoin
     redis_value_to_java(&mut env, *value, false)
 }
 
+#[no_mangle]
+pub extern "system" fn Java_glide_ffi_resolvers_RedisValueResolver_createLeakedBytesVec<
```

Comment:
Please add docs for new FFI methods. Unfortunately, we don't have them for other methods.

Reply:
Addressed in #1708